examples/kokkos-benchmarks/standalone/notype/bytes_and_flops.py

```diff
@@ -0,0 +1,108 @@
+import argparse
```

Comment:
is this a new example? let's keep this for the next PR and we focus this PR on removing types.